src/portable/extension.rs

```diff
@@ -53,27 +55,65 @@ fn get_local_instance(options: &Options) -> Result<InstanceInfo, anyhow::Error>
     Ok(inst)
 }
 
+#[tokio::main(flavor = "current_thread")]
```

Comment:
This seems out of place...

Reply:
Oh I see, this is being used to convert the function to a blocking one. I didn't know you could do this. It's not ideal -- I would recommend creating yourself a new tokio runtime and using block_on (see https://github.com/edgedb/edgedb-cli/blob/dbe9f20f5d584f3944edc2cf293209a7f78dff86/src/process.rs#L160)